code/modules/antagonists/paradox_clone/paradox_clone.dm

```diff
@@ -0,0 +1,116 @@
+/datum/antagonist/paradox_clone
+	name = "\improper Paradox Clone"
+	roundend_category = "Paradox Clone"
+	job_rank = ROLE_PARADOX_CLONE
+	prevent_roundtype_conversion = FALSE
```

Comment:
This var does nothing, so it doesn't do anything

![image](https://user-images.githubusercontent.com/53777086/206041578-54366a1f-5ce3-4d13-9d4d-cac72860acb7.png)

Someone should really delete it, it looks like a remnant of pre-dynamic